loader_hub/google_drive/base.py

```diff
@@ -132,35 +132,67 @@ def _get_fileids_meta(
                     mime_query = " or ".join([f"mimeType='{mime_type}'" for mime_type in mime_types])
                     query += f" and ({mime_query})"
 
-                results = service.files().list(q=query, fields="*").execute()
+                results = service.files().list(
+                    q=query, 
+                    includeItemsFromAllDrives=True,
+                    supportsAllDrives=True, 
+                    fields="*"
+                ).execute()
                 items = results.get("files", [])
                 for item in items:
                     if item["mimeType"] == folder_mime_type:
                         fileids_meta.extend(
                             self._get_fileids_meta(folder_id=item["id"], mime_types=mime_types)
                         )
                     else:
-                        fileids_meta.append(
-                            (
-                                item["id"],
-                                item["owners"][0]["displayName"],
-                                item["name"],
-                                item["createdTime"],
-                                item["modifiedTime"],
+                        # Check if file doesn't belong to a Shared Drive. "owners" doesn't exist in a Shared Drive
+                        is_shared_drive = 'driveId' in item
+                        if is_shared_drive:
+                            fileids_meta.append(
```

Comment:
to clarify, my overall point here is that tuples to fileids_meta should be the same length - it's just cleaner when reasoning about typing. if this is a matter of reasoning whether `item["owners"][0]["displayName"]` exists, then you can do something like `display_name = item["owners"][0]["displayName"] if is_shared_drive or None`, and pass that in to the tuple.

Reply:
Yes, that makes sense. I've pushed a new commit